platform/qt/src/qmapboxgl.cpp

```diff
@@ -1062,6 +1314,15 @@ void QMapboxGL::setFilter(const QString& layer_, const QVariant& filter_)
     qWarning() << "Layer doesn't support filters";
 }
 
+/*!
+    Renders the map using OpenGL draw calls. If \a fbo is passed, it will
+    make sure to bind the framebuffer object before drawing, otherwise a
+    valid GL context is expected with an appropriated GL viewport state set
+    for the size of the canvas.
+
+    This function should be called only after the signal needsRendering() is
```

Comment:
needsRendering() gets detected and linked by `qtdoc` automatically because it is a signal of the same class.